apps/nova/src/components/dashboard/PastLessons.tsx

```diff
@@ -74,6 +104,9 @@ export const PastLessons = () => {
         loading={lessonsQuery.query.isPending}
         error={lessonsQuery.query.isError}
         retry={lessonsQuery.query.refetch}
+        userRole={user?.role}
```

Comment:
The user role has more information than what the component needs. It has information about admins and sutdios for examples. You can replace it with a simple prop like `isTutor`. It is much better than accepting the entire user role enum. 


```suggestion
        isTutor={user?.role !== IUser.Role.Student}
```